lib/pages/surveydetail/widget/survey_question.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/gen/colors.gen.dart';
+import 'package:flutter_survey/models/question.dart';
+import 'package:flutter_survey/pages/surveydetail/widget/survey_answer.dart';
+import 'package:flutter_survey/resources/dimens.dart';
+
+class SurveyQuestion extends StatelessWidget {
+  final DisplayType displayType;
+
+  SurveyQuestion({required this.displayType});
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildPage(context);
+  }
+
+  Column _buildPage(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const SizedBox(height: Dimens.defaultMarginPaddingLarge),
+        Row(
+          children: [
+            Expanded(child: SizedBox.shrink()),
+            Padding(
+              padding: const EdgeInsets.only(left: Dimens.defaultMarginPadding),
+              child: GestureDetector(
+                onTap: () {
+                  // TODO https://github.com/luongvo/flutter-survey/issues/19
+                },
+                child: SizedBox(
+                  width: 56,
+                  height: 56,
+                  child: Assets.icons.icClose.svg(
+                    fit: BoxFit.none,
```

Comment:
@luongvo I think it is redundant 🤔

Reply:
@doannimble No, its touching size is 56, so `BoxFit.none` will help to scale it in its actual size instead of expanding to fit the box.